.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: DevTools issues
+    url: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/contact
+    about: To submit feature requests or bug reports about Microsoft Edge DevTools, contact the DevTools team.
```

Comment:
It's unclear what the Open button does, for DevTools.  In contrast, it's clear what the Open button does for WebView2: it opens that repo.